src/aks-preview/HISTORY.md

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.4.43
++++++
+* Add support for VMSS node public IP.
+
 0.4.38
```

Comment:
Where is 0.4.39 ~ 0.4.42?

Reply:
In setup.py file, the version is bumped from 0.4.42 to 0.4.43. I do not know why there is no release notes added here, what I do is to match to the latest version.